frontend/src/pages/locations/locations.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Button } from "@mui/material";
+import { useNavigate } from "react-router";
+import { LocationCard } from "components/molecules";
+import { DummyLocation1 } from "mocks";
+import { Location } from "types";
+import Page from "../page";
```

Comment:
can you sort the imports? first external, second internal (types, mocks ...), third atoms, molecules, organisms, and then pages